src/System.Private.ServiceModel/tools/scripts/OpenFirewallPorts.cmd

```diff
@@ -0,0 +1,18 @@
+@echo off 
+setlocal
+
+REM 
+REM Start Open all ports WCF test services use
+REM
+
```

Comment:
I'd prefer to avoid exe for readability reason. Let's investigate if there is a way to self elevating in a script.